src/drivers/SC7A20.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "AccelerationSensor.h"
```

Comment:
please use includes relative to `src`, otherwise I have a hard time maintaing the InfiniSim simulator ;)